packages/api/package.json

```diff
@@ -37,17 +37,17 @@
     "@nestjs/schematics": "^8.0.0",
     "@nestjs/testing": "^8.0.0",
     "@types/express": "^4.17.13",
-    "@types/jest": "27.0.2",
+    "@types/jest": "26.0.14",
     "@types/node": "^16.0.0",
     "@types/supertest": "^2.0.11",
     "eslint": "^8.0.1",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-prettier": "^4.0.0",
-    "jest": "^27.2.5",
+    "jest": "26.6.0",
```

Comment:
Nit but don't we want to be using the latest version of jest

Reply:
I was getting package resolution errors with CRA/React scripts when the versions were mismatched. There was an option to skip preflight checks and ignore the warnings which I admittedly don't fully understand the consequences of.

I valued consistency in versions across packages over having the latest version, what do you think?

Reply:
It should be fine, since this is a dev dependency. This does add some potential tech debt if we want to update our packages in the future.